### PR TITLE
remove ≔ from the language and replace it by :=

### LINKED ISSUE #1465: Remove usages of  ≔ and replace those with :=
(originally by @james-chf)

> the difference between ≔ and = is hard to spot, I thought = was the symbol to use when writing function clauses

### PR DESCRIPTION
**Pull request by janmasrovira**
closes #1465.

this pr also updates the stdlib to the latest version to reflect the syntax change.

### CODE REVIEW

File: juvix-mode/juvix-input.el [hunk 1]
```diff
@@ -229,7 +229,7 @@ order for the change to take effect."
 
   ;; Equality and similar symbols.
 
-  ("eq"  . ,(juvix-input-to-string-list "=∼∽≈≋∻∾∿≀≃⋍≂≅ ≌≊≡≣≐≑≒≓≔≕≖≗≘≙≚≛≜≝≞≟≍≎≏≬⋕"))
+  ("eq"  . ,(juvix-input-to-string-list "=∼∽≈≋∻∾∿≀≃⋍≂≅ ≌≊≡≣≐≑≒≓:=≕≖≗≘≙≚≛≜≝≞≟≍≎≏≬⋕"))
```
Comment (lukaszcz) [inline juvix-mode/juvix-input.el:232]: Is this intended or did it get changed by mistake?

File: juvix-mode/juvix-input.el [hunk 2]
```diff
@@ -244,7 +244,7 @@ order for the change to take effect."
   ("=="   . ("≡"))  ("==n" . ("≢"))
   ("==="  . ("≣"))
   (".="   . ("≐"))  (".=." . ("≑"))
-  (":="   . ("≔"))  ("=:"  . ("≕"))
+  (":="   . (":="))  ("=:"  . ("≕"))
```
Comment (lukaszcz) [inline juvix-mode/juvix-input.el:247]: As above?

Comment (janmasrovira) [reply juvix-mode/juvix-input.el:247]: mistake

Comment (janmasrovira) [reply juvix-mode/juvix-input.el]: fixed

**Review by lukaszcz — APPROVED** [review]

**Review by lukaszcz — DISMISSED** [review]

**Review by paulcadman — APPROVED** [review]

**Pull request merged**